packages/cf.js/src/provider.ts

```diff
@@ -10,17 +11,25 @@ import * as uuid from "uuid";
 
 import { AppFactory } from "./app-factory";
 import { AppInstance } from "./app-instance";
-import { AppDefinition } from "./structs";
+import { AppDefinition, AppInstallProposal } from "./app-types";
 
 export enum CounterfactualEventType {
   INSTALL = "cf_install",
   PROPOSE_INSTALL = "cf_proposeInstall",
   REJECT_INSTALL = "cf_rejectInstall"
 }
 
+export interface InstallProposalData {
+  proposal: AppInstallProposal;
+
+  reject();
```

Comment:
?

Reply:
Yeah, we were thinking of making rejectProposal a callback on the event. Probably not a good idea. Likely this is going to be changed anyway after sync with the Playground folks